gradle/SourceTemplate/app/build.gradle

```diff
@@ -0,0 +1,43 @@
+apply plugin: 'com.android.application'
+
+
+buildscript {
+    repositories {
+        jcenter()
+    }
+    dependencies {
+        // other classpath definitions here
+        classpath 'com.neenbedankt.gradle.plugins:android-apt:1.8'
+    }
+}
+
+android {
+    compileSdkVersion 25
```

Comment:
Oh, this is just a harness to make the template build in its own. Only the `src` folder of the template is actually copied to the projects.